mobile/src/main/java/com/gelakinetic/mtgfam/helpers/DecklistHelpers.java

```diff
@@ -222,7 +230,6 @@ public static String getSharableDecklist(
          */
         public CompressedDecklistInfo(MtgCard card, boolean isSideboard) {
```

Comment:
Should this constructor be replaced with a super now that it now that CompressedCardInfo extends MtgCard?